### DISCUSSION TITLE
Input focus out issue

### DISCUSSION THREAD

**Discussion by mohammadashfan24** [ritikbanger/react18-input-otp#9] [Q&A] [answered]
The focus does not move out of the input when enter key is pressed.  Can anyone help me with this issue?

**Comment by ritikbanger** [comment, marked as answer]
Can you create a reproducable example? Or a video example.

**Reply by ritikbanger** [reply]
We have an `onSubmit` prop that triggers when you press the `enter` key.

**Reply by mohammadashfan24** [reply]
I tried using a ref, it worked
